jax/_src/scipy/special.py

```diff
@@ -44,14 +44,16 @@ def gammaln(x: ArrayLike) -> Array:
   return lax.lgamma(x)
 
 
+def _gamma_sign(x: Array) -> Array:
+  floor_x = lax.floor(x)
+  return jnp.where((x > 0) | (x == floor_x), 1.0, (-1.0) ** floor_x)
```

Comment:
Nit:
```suggestion
  return jnp.where((x > 0) | (x == floor_x) | floor_x % 2 == 0, 1.0, -1.0)
```
avoids power operation.

Reply:
Done